test/test_uops.py

```diff
@@ -100,8 +100,11 @@ def _test_top_fxn(self, op, fxn, dts=(dtypes.float32, )*3):
 
 class TestFloatUOps(TestUOps):
   def test_neg(self): self._test_uop_fxn(UnaryOps.NEG, lambda a: -a)
+  @unittest.skipIf(Device.DEFAULT in {'CLANG', 'LLVM'}, 'only fastmath supported')
```

Comment:
LLVM?

Reply:
fixed, wasn't really paying attention to what i was writing